src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -159,6 +170,26 @@ private void runGcloudCommand(List<String> args, String... topLevelCommand)
     processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  private String runSynchronousGcloudCommand(List<String> args)
```

Comment:
I omitted the javadoc since this method is private, but I'll add some regular comments explaining.